addons/bus/tests/test_assetsbundle.py

```diff
@@ -25,7 +25,7 @@ def patched_sendone(self, channel, notificationType, message):
             if notificationType == 'bundle_changed':
                 sendones.append((channel, message))
 
-        self.patch(type(self.env['bus.bus']), '_sendone', patched_sendone)
+        self.patch(type(self.env['bus.bus']), '_add_to_queue', patched_sendone)
```

Comment:
Not directly related to this PR but this test should be adapted to use standard bus helpers and/or check the result on the bus table rather than using a patch.